client/src/components/App.js

```diff
@@ -33,15 +29,14 @@ const App = () => {
                 <Route path="/auth" exact component={Auth} />
                 <Route path="/register" exact component={Register} />
                 <Route path="/login" exact component={Login} />
-                <Route path="/editor" exact component={Editor}/>
-                
-                <Layout sidebarData={SidebarData} headerData={headerData} >
-                    <Route path="/publications/team" exact component={PublicationPage}/>
-                    <Route path="/dashboard" exact component={Dashboard} />
-                    <Route path="/dashboard/profile" exact component={ProfileInfoEdit} />
-                </Layout>
-
-            </Switch>             
+
+                <DashboardLayoutRoute path="/dashboard" exact component={Dashboard} />
+                <DashboardLayoutRoute path="/dashboard/profile" exact component={ProfileInfoEdit} />
+                <DashboardLayoutRoute path="/publications/team" exact component={PublicationPage} />
+
+                <EditorLayoutRoute path="/editor" exact component={EditorHome} />
```

Comment:
DashboardLayoutRoute is the route with dashboard layout, EditorLayoutRoute is with EditorLayout